### PR TITLE
Lastic Coretime Indexer + Visualizations

### PR DESCRIPTION
**Pull request by poppyseedDev**
#### Project Abstract

Our project, "Indexing Coretime Chain with Subsquid and Graph Implementation," is designed to enhance data analysis in the Polkadot ecosystem. We're focusing on indexing the Coretime Chain for precise data capture and creating interactive graphs for insightful chain performance visualization. As a unique venture in the ecosystem, our experienced team is committed to improving the understanding and usability of Polkadot's blockspace.

Previous grant applications:
 - https://github.com/w3f/Grants-Program/pull/2111
 - https://github.com/w3f/Grants-Program/pull/2008

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ Polkadot (USDC & USDT) address in the application).
- [x] I am aware that, in order to receive a grant, I (and the entity I represent) have to successfully complete a KYC/KYB check.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Review by Noc2 — DISMISSED** [review]
Thanks for the update. I'm happy to go ahead with it and share it with the rest of the team.

**Comment by poppyseedDev** [comment]
> Could you change the filename to md? This way it is easier to read.

I am so sorry about this. I was writing the text elsewhere and then just copied it. 😅

**Review by BenWhiteJam — DISMISSED** [review]

**Review by keeganquigley — CHANGES_REQUESTED** [review]
Thanks for the application @poppyseedDev based on your previous submissions I'm in favor of supporting one last grant, but I have one minor comment:

- We no longer accept BTC for payment, here are our current [payment options](https://github.com/w3f/Grants-Program?tab=readme-ov-file#pencil-process). Would you be willing to accept USDC on Asset Hub instead?

File: applications/lastic-grant3.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
+# Lastic - Grant 3 - Indexing Coretime Chain with Subsquid and Graph Implementation
+
+- **Team Name:** Lastic
+- **Payment Address:** bc1q0d6pkmsvncshyeqvzzlcfjc7lkf4fuku7nsen7 (BTC)
```
Comment (poppyseedDev) [inline applications/lastic-grant3.md:4]:
```suggestion
- **Payment Address:** 16Jbt5zru6XdUx1jYRV8AZtGk76ujtUNdd1UVWVVkVvGZ31L (USDT)
```

**Review by poppyseedDev — COMMENTED** [review]
Is this an okay format for a Polkadot Asset Hub address?

**Comment by keeganquigley** [comment]
Thanks @poppyseedDev yep that looks good.

**Review by semuelle — CHANGES_REQUESTED** [review]
We [just started requiring KYC/KYB checks](https://grants.web3.foundation/docs/faq#what-is-kyckyb-and-why-do-i-have-to-provide-this-data) for all potential grantees. Could you please follow the instructions under [this link](https://in.sumsub.com/idensic/l/#/uni_JRqwo72Y5AjXaRDO)? Let me know if you have any questions or issues!

**Review by keeganquigley — APPROVED** [review]

**Review by takahser — APPROVED** [review]
Jumping in for @BenWhiteJam here.

**Review by BenWhiteJam — APPROVED** [review]

**Comment by semuelle** [comment]
Hey @poppyseedDev, if you want to proceed with this grant before setting up an entity, you can also have every benefactor of the grant go through KYC through this link instead: https://in.sumsub.com/idensic/l/#/uni_mjcB05gJZfh8dbr7

**Comment by poppyseedDev** [comment]
> https://in.sumsub.com/idensic/l/#/uni_mjcB05gJZfh8dbr7

Oh, okay nice I didn't know that option existed, I now was successfully able to register.

**Comment by semuelle** [comment]
Good to hear, @poppyseedDev. Did all benefactors of the grant perform KYC? If not, can you please share the link with them as well?

**Pull request merged**